app/assets/stylesheets/pictures.scss

```diff
@@ -0,0 +1,112 @@
+// Place all the styles related to the pictures controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+a:link    {color:#000;}
+a:visited {color:#000;}
+a:hover   {color:#000;}
+a:active  {color:#000;}
+
+body {
+  background-color: #F0F0F0;
+  font-family: sans-serif;
+}
+
+.login {
+  background-color: white;
+  border-color: #E0E0E0;
+  border-style: solid;
+  border-width: 1px;
+  margin: 10px auto;
+  margin-bottom: 10px;
+  margin-top: 10%;
+  max-width: 250px;
+  padding: 10px;
+  position: relative;
+}
+
+.login-heading {
+  margin: 0 auto;
+  margin-left: 5px;
+  max-width: 14%;
+  position: relative;
+}
+
+.heading {
+  margin: 30px auto;
+  max-width: 14%;
+  position: relative;
+}
+
+.timeline {
+  margin: 10px auto;
+  margin-bottom: 0;
+  margin-top: 0;
+  max-width: 604px;
+  padding: 20px auto;
+  position: relative;
+}
+
+.post {
+  background-color: white;
+  border-color: #E0E0E0;
```

Comment:
Color literals like `#E0E0E0` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#E0E0E0` should be written as `#e0e0e0`